lib/tasks/work_tasks.rake

```diff
@@ -25,4 +25,14 @@ namespace :work do
     end
   end
 
+  # AO3-6065
+  desc "Add 'Not Rated' to works missing a rating"
+  task(:not_rated => :environment) do
+    Work.find_each do |work|
```

Comment:
Since this is going to take a while, can we use `find_in_batches` and then print a progress indicator to `STDOUT` so we can see how things are progressing? (There are a bunch of examples of possible approaches in the other recent after tasks.)